### PR TITLE
Create page for user register

### PR DESCRIPTION
**Pull request by MilenaHtec**

### CODE REVIEW

**Review by casualme2 — APPROVED** [review]

File: frontend/src/Components/Login/Login.jsx [hunk 1]
```diff
@@ -1,6 +1,253 @@
...
+  const [showPassword, setShowPassword] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [successfulLogin, setSuccessfulLogin] = useState("");
+
+  const emailRef = useRef();
+  const passwordRef = useRef();
+
+  const navigate = useNavigate();
+
+<<<<<<< HEAD
```
Comment (marija-antic) [inline frontend/src/Components/Login/Login.jsx:21]: remove this

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
+import { Routes, Route } from "react-router-dom";
+import HomePage from "./Components/HomePage/HomePage";
+import Login from "./Components/Login/Login";
+import SignUp from "./Components/SignUp/SignUp";
+import ProtectedRoute from "./Components/Routes/ProtectedRoute";
+import RedirectRoute from "./Components/Routes/RedirectRoute";
 
 const App = () => {
-	return <Loader />;
+	const [accessToken, setAccessToken] = useState(localStorage.accessToken);
```
Comment (marija-antic) [inline frontend/src/App.jsx:10]: Wasn't this the change from previous PR? Check if you resolved conflicts correctly

File: frontend/src/Components/Login/Login.jsx [hunk 3]
```diff
@@ -1,6 +1,253 @@
...
+      .catch(({ response }) => {
+        if (response.status === 400) {
+          setErrorMsg("Incorrect username or password");
+        } else if (response.status === 0) {
+          setErrorMsg("Server Timeout");
+        }
+        setLoading(false);
+      });
+  };
+=======
```
Comment (marija-antic) [inline frontend/src/Components/Login/Login.jsx:49]: and this

File: frontend/src/Components/Login/Login.jsx [hunk 4]
```diff
@@ -1,6 +1,253 @@
...
+						<p className="m-0">Don't have an account? </p>
+						<Link to={ROUTES.SIGNUP} href="#!" className="m-0 btn btn-link">
+							Register here
+						</Link>
+					</div>
+				</form>
+			</div>
+		</div>
+	);
+>>>>>>> bb56fdbf0b0ee29adbe25b3f2f8847dcbe5caab2
```
Comment (marija-antic) [inline frontend/src/Components/Login/Login.jsx:250]: ⬆️

**Pull request merged**